### PR TITLE
Feat/add spine performance

### PR DESCRIPTION
**Pull request by johanzhu**

### CODE REVIEW

**Comment by zhuxudong** [comment]
merge to dev/0.4

File: demos/spine-animation/index.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ const rootEntity = scene.createRootEntity();
 // camera
 const cameraEntity = rootEntity.createChild("camera_node");
 const camera = cameraEntity.addComponent(Camera);
-cameraEntity.transform.position = new Vector3(0, 0, 70);
+cameraEntity.transform.position = new Vector3(0, 0, 60);
```
Comment (GuoLei1990) [inline demos/spine-animation/index.ts:15]: use `setPosition()`

File: demos/spine-performance/index.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { Camera, Logger, SystemInfo, Vector3, WebGLEngine, Entity } from "oasis-engine";
+import { SpineAnimation } from "@oasis-engine/spine";
+import "@oasis-engine/stats";
+
+Logger.enable();
```
Comment (GuoLei1990) [inline demos/spine-performance/index.ts:5]: Delete this

**Comment by GuoLei1990** [comment]
avatar.png  is still need? @zhuxudong

**Review by GuoLei1990 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**